include/datadog_agent_six.h

```diff
@@ -48,6 +48,10 @@ DATADOG_AGENT_SIX_API int run_simple_string(const six_t *, const char *code);
 DATADOG_AGENT_SIX_API int has_error(const six_t *);
 DATADOG_AGENT_SIX_API const char *get_error(const six_t *);
 
+// PYTHON HELPERS
+DATADOG_AGENT_SIX_API char **get_integration_list(six_t *);
```

Comment:
We could use JSON here like we do in some of the builtins and simply return a `char*` so we don't need `free_integration_list`